tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_optimizer.py

```diff
@@ -41,14 +41,17 @@
 from tensorflow.python.training.tracking import base as trackable
 
 
-def DynamicEmbeddingOptimizer(self, bp_v2=None):
+def DynamicEmbeddingOptimizer(self, bp_v2=False, horovod_synchronous=False):
```

Comment:
The name of `horovod_synchronous` may bind the API to horovod only. I think `synchronous` might be better.